client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -242,7 +242,9 @@ const initialData: EditorProperties = {
 
 const Editor = (props: EditorProps): JSX.Element => {
   const { t } = useTranslation();
-  const { editorRef, initTests, resetAttempts } = props;
+  const propsRef = useRef<EditorProps>(props);
+  console.log(propsRef.current);
```

Comment:
```suggestion
  const propsRef = useRef<EditorProps>(props);
  const propsRef.current = props;
```
The ref approach is all a bit hacky, but the idea is that every render should update `propsRef` and so everything that uses it will have access to the most up-to-date prop values.